src/style.css

```diff
@@ -0,0 +1,66 @@
+main {
+  box-shadow: 12px 12px 12px 9px silver;
+  width: 45% !important;
+  margin-top: 12%;
+  padding: 4px 30px;
+}
+
+#list,
+#todays {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  border-bottom: 0.5px solid darkgray;
+  padding: 12px;
+  box-shadow: 2px 2px 2px darkgrey;
+}
+
+#list span {
+  font-size: 15px;
+}
+
+#defaultCheck1 {
+  border-color: black;
+  margin-right: 12px;
+}
+
+input {
+  height: 3rem;
+  border: none;
+  border-bottom: 0.5px solid darkgray;
+  box-sizing: border-box;
+}
+
+#input,
+.input-group {
+  padding: 0;
+  margin: 0;
+}
+
+input:focus {
+  outline: none;
+}
+
+input::placeholder {
+  font-style: italic;
+  font-size: 21px;
+}
+
+#footer {
+  padding: 21px;
+  background-color: rgb(246, 244, 243);
+}
+
+#footer p {
+  text-align: center;
+  font-size: 25px;
+}
+
+#litsBtn,
+#clearDones,
+#defaultCheck,
+#dots,
+.fa-ellipsis-v,
+.fa-trash {
+  cursor: pointer;
+}
```

Comment:
- [ ] While you did a wonderful job styling your project :+1:, as a [project requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md), Your list should be a clone of the part of the minimalist project captured in the [video](https://www.youtube.com/watch?v=AcUd-_Yjjqg).

**Your design**

![Screenshot from 2023-02-22 07-36-08](https://user-images.githubusercontent.com/105117832/220622137-1ff9febc-d0d4-442a-8b5b-c87a23b4ea18.png)

**Template**

![Screenshot from 2023-02-22 07-36-34](https://user-images.githubusercontent.com/105117832/220622195-4457107a-e04a-46d8-afa7-a73ee7c52e5b.png)